bridges/RadioMelodieBridge.php

```diff
@@ -0,0 +1,34 @@
+<?php
+class RadioMelodieBridge extends BridgeAbstract {
+	const NAME = 'Radio Melodie Actu';
+	const URI = 'https://www.radiomelodie.com/';
+	const DESCRIPTION = 'Retourne les actualités publiées par Radio Melodie';
+	const MAINTAINER = 'sysadminstory';
+	const PARAMETERS = array();
+	const CACHE_TIMEOUT = 3600;
+
+	public function collectData(){
+		// Implement your bridge here!
```

Comment:
This comment was intended to guide you during the creation of a bridge, it can be removed now.